# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======
#goal definition
float64[3] finger
float64 spread
float64 fingerVel
float64 spreadVel
bool fingerHold
bool spreadHold
